### DISCUSSION TITLE
AI與歷史的應用

### DISCUSSION THREAD

**Discussion by xipeiguo** [1101-GS4719-C/Python-Community-GS4719-C#2078] [General]
https://www.cdns.com.tw/articles/1072209
成功大學的歷史系在今年的個人申請中出現了沒人錄取的情形，備受熱議，在科技發展的過程中，純文類的科系似乎漸漸走上淘汰的命運，然而我認為歷史依舊能和AI互相合作，就如同本文所提，而應用的範圍我認為也很大，例如資料庫的累積等等，人們在重視AI的過程中，若能與純文類科系進行合作，想必能創造一個更友善、包容的社會。

**Comment by peresonacifer** [comment]
文組在台灣的待遇原本就不是很好，如今AI發展迅速，更出現某些科系沒有人要念的情況，但我認為如果是真的有興趣想念還是可以念，但可能需要學習試著跟AI做結合，例如嘗試把AI運用在歷史方面，我認為這種應用方面的人才會是需要的。

**Comment by ds0w0** [comment]
老實說我認為台灣過度仰賴科技業而忽視了文組的價值和重要性，AI在搜索和整理資料方面有很大的優勢，我認他能更好的質化和量化文本，我們可以透過它來分析或著探索史料和文獻，加深我們對於一個文化的認識，更能利用AI來推測或演繹特定主題，來幫助我們吸收科系內所學知識。

**Comment by 113607514** [comment]
最近，成功大學歷史系在個人申請中沒有人錄取的消息引起了不少關注。隨著科技的進步，純文科系似乎面臨越來越大的挑戰，甚至有些人開始擔心它們會被淘汰。但我其實認為，歷史仍然能在 AI 的世界中找到自己的位置。比如，在資料庫的累積和歷史資料的整理上，AI 可以大大提高效率，幫助我們更深入地理解過去，從中汲取智慧。這樣的合作不僅能保留歷史學科的價值，還能讓它在現代社會中發揮更大的作用。如果我們能讓 AI 和純文科系攜手合作，相信能創造出一個更加包容和多元的未來，讓不同領域的知識交織出更多可能性。
